netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -1204,6 +1224,21 @@ net_ebpf_extension_sock_addr_authorize_connection_classify(
     return;
 }
 
+static BOOLEAN
+_net_ebpf_ext_sock_addr_is_connection_locally_redirected_by_others(
+    _In_ const FWPS_CONNECT_REQUEST* connect_request, uint64_t filter_id)
+{
+    FWPS_CONNECT_REQUEST* previous_connect_request = connect_request->previousVersion;
+    while (previous_connect_request != NULL) {
+        if (previous_connect_request->modifierFilterId != filter_id) {
+            if (previous_connect_request->localRedirectHandle != NULL)
```

Comment:
nit: This could be merged into a single if statement (though I do see that our public docs has them separate). Certainly not a blocking comment though.